blocks/product-details/product-details.js

```diff
@@ -151,25 +151,30 @@ export default async function decorate(block) {
   let inlineAlert = null;
   const routeToWishlist = '/wishlist';
 
-  async function renderRequisitionListNamesIfEnabled($container) {
+  async function renderRequisitionListNamesIfEnabled($container, currentOptions = undefined) {
```

Comment:
being `undefined` sound more uncontrolled.
I think it would be more appropriate to use `null`